src/plugins/intel_cpu/tests/functional/subgraph_tests/src/eltwise_caching.cpp

```diff
@@ -32,23 +32,23 @@
 //                                   |output|
 //                                   --------
 
-#include <tuple>
-#include <string>
-#include <vector>
-#include <memory>
-#include <debug.h>
-#include <shared_test_classes/base/ov_subgraph.hpp>
-#include <ov_models/builders.hpp>
 #include "common_test_utils/common_utils.hpp"
-#include <common_test_utils/ov_tensor_utils.hpp>
-#include "functional_test_utils/skip_tests_config.hpp"
+#include "common_test_utils/node_builders/eltwise.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "ov_models/builders.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
 #include "test_utils/cpu_test_utils.hpp"
 
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
 using namespace CPUTestUtils;
-using ngraph::helpers::EltwiseTypes;
-using namespace ov::test;
 
-namespace CPUSubgraphTestsDefinitions {
+namespace ov {
+namespace test {
+using namespace ov::test::utils;
```

Comment:
It just to avoid adding too much ov::test::utils or utils prefix before some object, such as:
some need change to `ov::test::utils::EltwiseTypes` still can keep it as `EltwiseTypes`